<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDlg</class>
 <widget class="QDialog" name="ConnectDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>201</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connect To Erebus Server</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../erebus.qrc">
    <normaloff>:/images/logo32.png</normaloff>:/images/logo32.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="dialogLayout" stretch="0,0,0">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelEndpoint">
       <property name="text">
        <string>Endpoint</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboEndpoints">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
       <property name="duplicatesEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelUser">
       <property name="text">
        <string>User</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editUser"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editPassword">
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkSsl">
       <property name="text">
        <string>SSL</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelRootCA">
       <property name="text">
        <string>Root CA</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="editRootCA"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnBrowseRootCA">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="dlgButtonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../erebus.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>ConnectDlg</receiver>
   <slot>onOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>ConnectDlg</receiver>
   <slot>onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBrowseRootCA</sender>
   <signal>clicked()</signal>
   <receiver>ConnectDlg</receiver>
   <slot>onBrowseRootCA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkSsl</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ConnectDlg</receiver>
   <slot>onSsl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOk()</slot>
  <slot>onCancel()</slot>
  <slot>onSsl()</slot>
  <slot>onBrowseRootCA()</slot>
 </slots>
</ui>
